go.mod

```diff
@@ -6,9 +6,13 @@ require (
 	github.com/RHsyseng/operator-utils v1.4.2
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
-	github.com/go-logr/logr v0.3.0
-	github.com/imdario/mergo v0.3.10
-	github.com/kubernetes-csi/external-snapshotter/v2 v2.1.2
+	github.com/go-logr/logr v0.4.0
+	github.com/gogo/protobuf v1.3.2 // indirect
+	github.com/google/go-cmp v0.5.4 // indirect
+	github.com/google/gofuzz v1.2.0 // indirect
+	github.com/googleapis/gnostic v0.5.3 // indirect
+	github.com/imdario/mergo v0.3.11
+	github.com/kubernetes-csi/external-snapshotter/client/v4 v4.0.0
```

Comment:
this should be 4.1.0? which is used with    OCP 4.8

Reply:
Good catch, looks like something happened after running `go mod vendor`, let me check